packages/starship/0001-Cargo.toml.patch

```diff
@@ -2,21 +2,10 @@
 +++ b/Cargo.toml
 @@ -29,8 +29,7 @@
  """
- 
+
  [features]
--default = ["battery", "notify", "git-repository-max-perf"]
+-default = ["battery", "notify", "gix-max-perf"]
 -battery = ["starship-battery"]
-+default = ["notify", "git-repository-max-perf"]
++default = ["notify", "gix-max-perf"]
  config-schema = ["schemars"]
  notify = ["notify-rust"]
- 
```

Comment:
If the dependency is as optional and no feature uses it then it does not need to be removed because it will not be used anyway. (This would make a conflict for most updates since dependency versions change)

Reply:
bruh